web/controllers/user_controller.ex

```diff
@@ -8,6 +8,7 @@ defmodule CodeCorps.UserController do
 
   def index(conn, _params) do
     users = Repo.all(User)
+    users = Repo.preload(users, [:slugged_route])
```

Comment:
```Elixir
users =
  User
  |> preload([:slugged_route])
  |> Repo.all
```

Should do the trick, but I'm pulling this out of my head, without trying it out.